src/components/InputSet/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Input } from '../../variables';
+import { User } from '../../Shared';
+import styles from './styles.module.css';
+
+interface InputsStatesSignature extends User {
```

Comment:
This interface presents at 2 files. Please check.

Reply:
removed, thanks